prov/tcp/src/tcpx_cq.c

```diff
@@ -60,6 +60,39 @@ static int tcpx_cq_close(struct fid *fid)
 	return 0;
 }
 
+static void tcpx_xfer_entry_reset(struct tcpx_buf_pool *pool,
+                                  struct tcpx_xfer_entry *xfer_entry)
+{
+	memset(xfer_entry, 0, sizeof(*xfer_entry));
```

Comment:
This is a large structure that we really don't want to re-initialize every single time it is used.  Most of the fields will be set further on in the code.